setup.cfg

```diff
@@ -0,0 +1,10 @@
+[flake8]
+max-line-length = 90
+max-complexity = 10
+statistics = true
+application-import-names = user_deletion
+import-order-style = smarkets
+
+[coverage:run]
+source = user_deletion
+omit = *tests*
```

Comment:
Added in https://github.com/incuna/django-user-deletion/pull/1/commits/f88d25d881db037a6c9f5e64e7861b55c00fe7b1